scenarios/kubernetes_e2e.py

```diff
@@ -586,7 +586,7 @@ def main(args):
         set_up_kops_aws(mode.workspace, args, mode, cluster, runner_args)
     elif args.deployment == 'kops' and args.provider == 'gce':
         set_up_kops_gce(mode.workspace, args, mode, cluster, runner_args)
-    elif args.gce_ssh:
+    elif args.gce_ssh and args.deployment != 'eks':
```

Comment:
don't set it?

Reply:
(aka set `--gce-ssh=""` in your job instead?)

Reply:
That's better. Will change this branch to do that. Thanks.